buildenv/jenkins/jobs/infrastructure/copyrightCheckDir.sh

```diff
@@ -0,0 +1,612 @@
+#!/bin/sh
+# ===========================================================================
+# (c) Copyright IBM Corp. 2018, 2019 All Rights Reserved
+# ===========================================================================
+#
+# This code is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 only, as
+# published by the Free Software Foundation.
+#
+# IBM designates this particular file as subject to the "Classpath" exception 
+# as provided by IBM in the LICENSE file that accompanied this code.
+#
+# This code is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+# version 2 for more details (a copy is included in the LICENSE file that
+# accompanied this code).
+#
+# You should have received a copy of the GNU General Public License version
+# 2 along with this work; if not, see <http://www.gnu.org/licenses/>.
+#
+# ===========================================================================
+
+trace () {
+  if [ "$VERBOSE" = "1" ]; then
+    echo $1
+  fi
+  return
+}
+
+log () {
+  echo $1 1>&2
+  return
+}
+
+check () {
+  if [ ! "$ROOTDIR" = "" ]; then
+    case "$1" in
+      $ROOTDIR/*) CHECK=1;;
+      *) return 0;;
+    esac
+  fi
+  trace "Checking file $1"
+
+  ERROR=0
+
+  CHECK=0
+  case "$1" in
+     # Comment out to check the test directory files.
+    test/*)
+      trace "Ignoring $1 because it appears to match case parameter expansion test/*"
+      CHECK=0;;
```

Comment:
Since we set `CHECK=0` above, is it necessary to also have it in all the cases?

Reply:
Not necessary, but I thought it made it clear?